src/features/game/types/bumpkinActivity.ts

```diff
@@ -19,11 +19,12 @@ type SellableName =
   | BeachBountyTreasure
   | FruitName;
 
-type Recipes = Food | ConsumableName;
+type Recipes = Food | CookableName;
+type Edibles = Food | ConsumableName;
 
 export type HarvestEvent = `${CropName | FruitName} Harvested`;
 export type CookEvent = `${Recipes} Cooked`;
-export type FedEvent = `${Recipes} Fed`;
+export type FedEvent = `${Edibles} Fed`;
```

Comment:
What happens to old `Recipes Fed` analytics?

Reply:
@adamhannigan I think it should be OK. The `Edibles` contain all the `Recipes`.

Reply:
I misunderstood, carry on!